include/circt/Dialect/RTL/RTLOps.h

```diff
@@ -58,11 +58,13 @@ StringAttr getModuleArgumentNameAttr(Operation *module, size_t argNo);
 StringAttr getModuleResultNameAttr(Operation *module, size_t argNo);
 
 static inline StringRef getModuleArgumentName(Operation *module, size_t argNo) {
-  return getModuleArgumentNameAttr(module, argNo).getValue();
+  auto attr = getModuleArgumentNameAttr(module, argNo);
+  return attr ? attr.getValue() : "";
```

Comment:
The returned attr can't be null here. Please don't add this defensive code or in the method below.